test/WebJobs.Script.Tests/WebScriptHostHandlerTests.cs

```diff
@@ -41,6 +41,7 @@ public WebScriptHostHandlerTests()
         public async Task SendAsync_HostNotRunning_Returns503()
         {
             _managerMock.SetupGet(p => p.IsRunning).Returns(false);
```

Comment:
If you make the suggested change below, we can then update this test to verify that IsRunning is called multiple times, verifying that the polling loop is polling.